{"uuid": "016ef8a7-caee-40a8-8676-fee0043302ab", "counter": 7, "stdout": "\u001b[1;31mfatal: [192.168.1.1]: UNREACHABLE! => {\"changed\": false, \"msg\": \"Failed to connect to the host via ssh: ssh: connect to host 192.168.1.1 port 22: Connection refused\", \"unreachable\": true}\u001b[0m", "start_line": 5, "end_line": 6, "runner_ident": "462d4d02-ae4b-4863-b65a-85525255480a", "event": "runner_on_unreachable", "pid": 4576, "created": "2022-04-22T13:05:24.369406", "parent_uuid": "9e73ed77-7fb2-5657-08a8-00000000000f", "event_data": {"playbook": "deploy.yaml", "playbook_uuid": "7b7049e6-b789-452b-a2a3-1fefa483cce6", "play": "this is a  hello world example", "play_uuid": "9e73ed77-7fb2-5657-08a8-000000000007", "play_pattern": "all", "task": "Gathering Facts", "task_uuid": "9e73ed77-7fb2-5657-08a8-00000000000f", "task_action": "gather_facts", "task_args": "", "task_path": "/Users/devops/data/project/python/pgoops/upload/iac/helloworld/project/deploy.yaml:2", "host": "192.168.1.1", "remote_addr": "192.168.1.1", "start": "2022-04-22T13:05:24.350830", "end": "2022-04-22T13:05:24.369340", "duration": 0.01851, "res": {"unreachable": true, "msg": "Failed to connect to the host via ssh: ssh: connect to host 192.168.1.1 port 22: Connection refused", "changed": false}, "uuid": "016ef8a7-caee-40a8-8676-fee0043302ab"}}